.github/workflows/ci_win.yml

```diff
@@ -29,6 +29,7 @@ jobs:
           - test_suite: samples
           - test_suite: api_coverage
           - test_suite: behavior_tests
+          - test_suite: applications
```

Comment:
We can add Windows applications test suite until there is a runnable Windows application.

Reply:
I think we can reserve the change here.

Reply:
Avoid others forgetting add test suit here if new case added.

Reply:
Done